src/projects/mesonproject.vala

```diff
@@ -783,6 +795,38 @@ class Vls.MesonProject : Project {
 
     public MesonProject (string root_path, FileCache file_cache, Cancellable? cancellable = null) throws Error {
         base (root_path, file_cache);
+        // read system envionment;
```

Comment:
Is this really needed? Doesn't the spawned meson subprocess inherit the environment variables from vls?

Reply:
I added it here for other reasons, I will remove it

Reply:
well, it depends on the reasons...